.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+* timjaczesko@google.com interweb@google.com corykramer@google.com
```

Comment:
I was able to update the permissions for this repo, so could we switch this back to the group
```
* @GoogleCloudPlatform/recaptcha-customer-obsession-reviewers
```